backend/tests/test_endpoints.py

```diff
@@ -3,7 +3,7 @@
 from rope.api.main import app
 from rope.api.sessions import get_request_session, session_store
 from rope.api.database import SessionLocal
-from rope.db.schema import UserAccount, SchoolDistrict, MoodleSetting
+from rope.db.schema import UserAccount, SchoolDistrict, MoodleSetting, CourseBuild
```

Comment:
Another "code smell" comment: This file is getting pretty long. Do we want to try and split these tests out in a way that makes the more manageable in length? If we take the router pattern, we can split out to align that way :man_shrugging: .